simplemind/providers/ollama.py

```
@@ -0,0 +1,63 @@
+    def client(self):
+        """The raw Ollama client."""
+        return ol.Client(
+            timeout=TIMEOUT,
+            host=self.host_url)
+
+    @property
+    def structured_client(self):
+        """A client patched with Instructor."""
+        raise NotImplementedError(NOT_IMPLEMENTED_REASON)
+
+    def send_conversation(self, conversation: "Conversation"):
```

Ruff

Undefined name `Conversation`

(F821)